protocol/development-workflow/README.md

```diff
@@ -1,171 +1,75 @@
 # Development Workflow
```

Comment:
I would argue the title could be "Story Development Workflow", or something like that. Because development workflow, to me, just means development, and NOT QA, whereas mentioning the story means we want to complete the lifecycle of the story.